src/app/CommandHandler.h

```diff
@@ -286,7 +286,7 @@ class CommandHandler
     template <typename CommandData>
     void AddResponse(const ConcreteCommandPath & aRequestCommandPath, const CommandData & aData)
     {
-        DataModel::EncodableType<CommandData> encodable(aData);
+        DataModel::EncodableType<CommandData> encodable(aData); // Fails
```

Comment:
The issue is this guy; it's using EncodableType, which is used both for commands & structs...

Reply:
But aData here is never a struct; it's always a command payload.

Reply:
Where is EncodableType used for structs?   I only see it being used for command payloads....